python/cudf/cudf/tests/test_groupby.py

```diff
@@ -2698,14 +2698,15 @@ def test_groupby_pct_change_empty_columns():
         False,
     ],
 )
-def test_groupby_group_keys(group_keys):
+@pytest.mark.parametrize("by", ["A", ["A", "B"]])
+def test_groupby_group_keys(group_keys, by):
     gdf = cudf.DataFrame(
-        {"A": "a a b".split(), "B": [1, 2, 3], "C": [4, 6, 5]}
+        {"A": "a a b".split(), "B": [1, 1, 3], "C": [4, 6, 5]}
```

Comment:
Do we need to have only two groups here? I'd like to see some cases where the same `A` value is used by groups with different `B` values. Something like:
```suggestion
        {"A": "a a a a b b".split(), "B": [1, 1, 2, 2, 3, 3], "C": [4, 6, 5, 9, 8, 7]}
```
which has 3 groups of 2 rows, with `("a", 1), ("a", 2), ("b", 3)`.